core/include/detray/core/detector.hpp

```diff
@@ -35,10 +37,14 @@
 namespace detray {
 
 /// @brief Forward declaration of a detector view type
-template <typename metadata, template <typename> class bfield_t,
-          typename container_t>
+template <typename bfield_t, typename detector_t>
 struct detector_view;
 
+/// Global definition of covfie backend for const bfield
+using const_backend_t =
```

Comment:
Let's not make this a global typedef in the detector that other headers rely on. I would say we either define it where we need it or we but the covfie types detray uses in their own header maybe? There is no guarantee this will always be here. Also I would rather call it ```const_bfield_bknd_t``` (even though it a bit unwieldy), since detray also has other backends

Reply:
Sounds good, I'll make the change